project.clj

```diff
@@ -1,15 +1,15 @@
-(defproject clojider-gatling-highcharts-reporter "0.3.2"
+(defproject clojider-gatling-highcharts-reporter "0.3.3"
```

Comment:
This is quite a big change. It might make sense to bump version to `0.4.0`

Reply:
👍 Done